android/res/layout/list_separator.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="fill_parent"
+    android:layout_height="wrap_content"
+    android:background="#AAAAAA"
```

Comment:
OK, for now, but aren't there standard colors?

Reply:
What do you mean?